guide/react-native/background.mdx

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 6
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Streaming from background
+
+On Android, it is possible to continue streaming when app is in background. Unfortunately this functionality is not available on iOS (due to Apple limitations)
```

Comment:
**nitpick:** Are you sure this is true? I think it's possible to achieve using CallKit (at least audio, this is what google meets does), however our app does not support it.

Reply:
Here is more investigation done by Angelika. We might work on this in future, to make it work somehow.